nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -173,20 +173,28 @@ def _create_statistics_aggregator(self,
             from nncf.experimental.onnx.statistics.aggregator import \
                 ONNXStatisticsAggregator
             return ONNXStatisticsAggregator(dataset)
+        if backend == BackendType.OPENVINO:
+            from nncf.experimental.openvino_native.statistics.aggregator import \
+                OVStatisticsAggregator
+            return OVStatisticsAggregator(dataset)
         return None
 
     def _apply(self,
                model: TModel,
                statistic_points: Optional[StatisticPointsContainer] = None,
                dataset: Optional[Dataset] = None) -> TModel:
+        backend = get_backend(model)
+        if backend == BackendType.OPENVINO:
```

Comment:
Have you checked with the nGraph team about the bug with the `deepcopy`? I mean, it's not good to have such if condition in the sharable part.

Reply:
ticket: 100919